### PR TITLE
chore(vue 3): shorten import path for vue3

### PR DESCRIPTION
**Pull request by eunjae-lee**
## Summary

This PR shortens the import path for vue3

from

```js
import InstantSearch from 'vue-instantsearch/dist/vue3/es';
```

to

```js
import InstantSearch from 'vue-instantsearch/vue3/es';
```

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -20,8 +20,7 @@ const processEnv = conf => ({
 });
 
 const vuePlugin = process.env.VUE_VERSION === 'vue3' ? vueV3 : vueV2;
-const outputDir =
-  process.env.VUE_VERSION === 'vue3' ? 'dist/vue3' : 'dist/vue2';
+const outputDir = process.env.VUE_VERSION === 'vue3' ? 'vue3' : 'dist/vue2';
```
Comment (Haroenv) [inline rollup.config.js:23]: should it not also be /vue2 then? instead of dist/vue2?

Comment (eunjae-lee) [reply rollup.config.js:23]: yeah let's go with that

**Comment by eunjae-lee** [comment]
The Netlify build failure is expected. (will be fixed later)

File: package.json [hunk 2]
```diff
@@ -18,15 +18,16 @@
...
   "files": [
-    "dist",
+    "vue2",
+    "vue3",
     "src"
   ],
-  "main": "dist/vue2/cjs",
-  "module": "dist/vue2/es",
+  "main": "vue2/cjs",
+  "module": "vue2/es",
```
Comment (Haroenv) [inline package.json:26]: was already the case, but why not vue2/cjs/index.js ?

File: package.json [hunk 3]
```diff
@@ -18,15 +18,16 @@
...
   "sideEffects": false,
   "repository": "https://github.com/algolia/vue-instantsearch",
   "scripts": {
-    "prebuild": "rm -rf dist",
+    "prebuild": "rm -rf vue2 && rm -rf vue3",
```
Comment (Haroenv) [inline package.json:30]:
```suggestion
    "prebuild": "rm -rf vue2 vue3",
```

**Review by Haroenv — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (eunjae-lee) [reply package.json:26]: any reason or concern not having `/index.js` in the path?

Comment (Haroenv) [reply package.json:26]: I guess it's supported, but I assumed that module and main were files, not module identifiers. If it works, then there's probably no issue

**Pull request merged**